src/binder/bind/read/bind_load_from.cpp

```diff
@@ -58,7 +58,19 @@ std::unique_ptr<BoundReadingClause> Binder::bindLoadFrom(const ReadingClause& re
                 throw BinderException(ExceptionMessage::variableNotInScope(objectName));
             }
         } else if (objectSource->objectNames.size() == 2) {
-            scanFunction = getObjectScanFunc(objectSource, clientContext);
+            // Bind external database table
```

Comment:
revert? We can reuse the getObjectScanFunction.

Reply:
Done.